api/src/main/java/com/avast/metrics/TimerPairImpl.java

```diff
@@ -54,4 +54,14 @@ public TimerPairImpl(Timer successTimer, Timer failureTimer) {
             throw ex;
         }
     }
+
+    @Override
+    public Timer getSuccessTimer() {
```

Comment:
You are right, I have reverted these changes in https://github.com/avast/metrics/pull/12.